README.md

```diff
@@ -1,2 +1,53 @@
-# Design-Teardown
-This project consists of building a heatmap of the Smashing magazine website
+# DESIGN TEARDOWN
+
+This project was developed as a learning project part of the Microverse curriculum.
+
+## Desktop
+![a gray-shade heatmap of the Smashing Magazine website.](./img/screenshot.PNG)
+
+## Built With
+
+- HTML5
+- CSS3
+- Google Fonts
+- Font awesome
+
+## Live Demo
+
+<a href= "https://rawcdn.githack.com/enkog/Design-Teardown/6d98b5c9306a1219663ea3424a5a56ef6fe86803/index.html" target="_blank">Live version</a>
+
+## Authors
+
+Prerequisites
+Webbrowser and access to internet
+
+Install
+[Download](Clone the repository to your machine)
+$ git clone https://github.com/enkog/Design-Teardown/tree/feature 
+Navigate to the project folder and execute the "index.html" file.
+Usage
+Just navigate through the page.
+
+Authors
```

Comment:
- [ ] Please, to have a [professional readme](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) check how your readme is displayed. Everything is smashed together and the ##Authors title is displaying things outside its section